Update base

luiz opened the pull request:
:heavy_check_mark:  @thiagocorbalan 

**Descrição**
Atualiza nossa lib de ordenação drag'n'drop, que usamos para ordenar coleções e produtos na tela, por exemplo. Nós fizemos um fork de uma lib opensource e modificamos algumas coisas, mas daí nossa versão ficou desatualizada com relação à versão oficial. Esse PR é para a gente atualizar nosso fork.

Commit:
(fix) Bind context to setTimeout

RubaXa committed:
Help wanted

Sapphire2k committed:
Update Sortable.js

timoschwarzer committed:
Fix deadlock when having a form with name=host

ibrahimbutt committed:
Fix README.md list formatting

gagaXD committed:
Fix event path for firefox

SturdyCat committed:
Modify the onChoose event firing order

How the onChoose event is fired before the sortable-chosen style is added, it should be able to do more fine control

owen-m1 committed:
toggle chosen class after events dispatched

owen-m1 committed:
Fixed indent

owen-m1 committed:
Merge pull request #1381 from ibrahimbutt/master

Fix README.md list formatting

owen-m1 committed:
Merge pull request #1384 from chinacoolboyzz/master

Modify the onChoose event firing order to allow for getting element information before chosenClass is applied

p-herbert committed:
(fix) Bind context directly to function

owen-m1 committed:
Merge pull request #1322 from p-herbert/master

(fix) Bind context to setTimeout

owen-m1 committed:
Added autoscroll revamp

In short, this revamp allows for elements that are parents of the currently hovered element to also autoscroll.

For native DnD, it autoscrolls the parents, but does not autoscroll the hovered element, as this is handled by HTML5.

owen-m1 committed:
Undo sortable.min.js modification

owen-m1 committed:
Merge pull request #1363 from timoschwarzer/master

Fix deadlock when having a form with name=host

owen-m1 committed:
fixed toFn issue 

Fixed issue in toFn causing checkPull and checkPut to always be true when `value == null`

owen-m1 committed:
Fix for scroll == HTML element

owen-m1 committed:
toFn redo

owen-m1 committed:
Updated toFn

Allow for 'clone' to be returned if value is 'clone'

owen-m1 committed:
Document allowing arrays in `put`

owen-m1 committed:
check names for equality instead of whole obj

owen-m1 committed:
Resolved PR #1338

owen-m1 committed:
Return if over el

yak80 committed:
Fix delayed dragging in Ios and Android

owen-m1 committed:
Modify instructions

owen-m1 committed:
added `bubbleScroll` option; added proper ability to cancel _throttle

owen-m1 committed:
added documentation

owen-m1 committed:
fix body autoscroll

owen-m1 committed:
Merge pull request #1386 from RubaXa/autoscroll-revamp

Autoscroll revamp

owen-m1 committed:
added dragoverBubble support for fallback

owen-m1 committed:
Merge pull request #1389 from yak80/fix-ios-android

Fix delayed dragging in Ios and Android

owen-m1 committed:
fixed owner detection; pull/put allow if same group

owen-m1 committed:
Merge branch 'master' of https://github.com/RubaXa/Sortable

owen-m1 committed:
Updated links

owen-m1 committed:
Merge pull request #1333 from Sapphire2k/master

Fix Android start dragging doesn't work

owen-m1 committed:
Merge pull request #1382 from saloonio/dev-firefoxShadowRoot

Fix event path for firefox

owen-m1 committed:
fixed clone rootEl mismatch and removed lastPullMode

owen-m1 committed:
undo 1d2beb514713f49840491bc94f42f07f8623b89e bc detects touch on PC

owen-m1 committed:
fix clone issue

owen-m1 committed:
Fix JSBins

owen-m1 committed:
1.8.0-rc1

owen-m1 committed:
CDN

owen-m1 committed:
fixed ghostEl dimensions

owen-m1 committed:
migration to gulp

owen-m1 committed:
Merge branch 'master' of https://github.com/SortableJS/Sortable

owen-m1 committed:
1.8.0

owen-m1 committed:
1.8.1

owen-m1 committed:
#1431: Remove pointercancel; #985: remove dropEffect

owen-m1 committed:
#1434: Target must be immediate child of sortable

owen-m1 committed:
added scroll adjustment

owen-m1 committed:
fixed detection for display:block

owen-m1 committed:
fixed detect direction for floats; revamped _ghostIsLast

owen-m1 committed:
#1436: Allow touchmove if not cancelable

owen-m1 committed:
empty sortable detection on touch move

owen-m1 committed:
always give ctx if includeCtx

owen-m1 committed:
1.8.2

owen-m1 committed:
#1446: fixed dropEffect

owen-m1 committed:
1.8.3

owen-m1 committed:
#1451: Detect grid direction

owen-m1 committed:
Added thanks to BrowserStack

owen-m1 committed:
#1458: Fixed autoscroll on MacOS Safari

owen-m1 committed:
Merge branch 'master' of https://github.com/SortableJS/Sortable

owen-m1 committed:
fixed fallback text selection on Safari

owen-m1 committed:
#659: Fixed delay in FireFox + Delay browser compatibility fixes

owen-m1 committed:
Merge commit 'f27342a2aa6cbfa16b4066a494e90107e334be36'

owen-m1 committed:
removed unnecessary _getScrollPosition function

David-Desmaisons committed:
Added pullMode to event object + added originalEvent to onStart

owen-m1 committed:
revert Sortable.min.js to last release

owen-m1 committed:
#1457: Improved detection of empty sortable

owen-m1 committed:
improvements to completed()

owen-m1 committed:
#1448: Position ghost absolutely on IOS

owen-m1 committed:
1.8.4

owen-m1 committed:
fixed field

owen-m1 committed:
#1471: Use real event properties when doing empty insert

owen-m1 committed:
#1475: Fixed _lastChild failing to ignore display:none

owen-m1 committed:
#1476: Stricter requirements for supportPointer

owen-m1 committed:
Better fix for unwanted IE event capturing

lehni committed:
Optimize nearestEmptyInsertDetectEvent callback (#1483)

MGelinas1 committed:
#1472: Added oldDraggableIndex + newDraggableIndex

owen-m1 committed:
#1487: Detect for empty insert always if not inserted

owen-m1 committed:
Merge branch 'master' of https://github.com/SortableJS/Sortable

owen-m1 committed:
#1489: Fix infinite loop in detecting shadow DOM

owen-m1 committed:
#1492: Improved repaint trigger

owen-m1 committed:
#1469: Fixed delay option on Android

seanogdev committed:
Add option to only apply delay on touch events (#1482)

owen-m1 committed:
1.9.0

lvegerano committed:
Add EmberJS addon to readme (#1523)

smnbbrv committed:
rename angularjs to angular (#1526)

owen-m1 committed:
Next version (#1524)

Aitthi committed:
Update Sortable.js

[ FIX BUG ] fix error in angular server side render

owen-m1 committed:
#1527: Added MultiDrag indicies event options

owen-m1 committed:
Update CONTRIBUTING.md

owen-m1 committed:
restore 1.9.0 _ghostIsLast

owen-m1 committed:
Merge pull request #1528 from Aitthi/patch-1

fix: error in angular server side render

owen-m1 committed:
restore 1.9.0 different row/col behaviour

owen-m1 committed:
Updated demo

owen-m1 committed:
1.10.0-rc1

owen-m1 committed:
correct plugin mounting examples

owen-m1 committed:
Better plugin docs

owen-m1 committed:
Fixed some ESM issues

owen-m1 committed:
1.10.0-rc2

owen-m1 committed:
Better IE compatibility

owen-m1 committed:
#1517: Mark Chrome for Android as not supporting native DnD

owen-m1 committed:
minor performance improvement

owen-m1 committed:
#1539: Better loop safety

owen-m1 committed:
Mark IOS as not supporting native drag & drop

owen-m1 committed:
1.10.0-rc3

owen-m1 committed:
#1556: Fix delayOnTouchOnly for android

owen-m1 committed:
Thanks to SauceLabs

owen-m1 committed:
Add tests

owen-m1 committed:
Fix CircleCI for PRs

beljand committed:
Fixed missing forEach indexes + Fixed CircleCI Config

owen-m1 committed:
#1580: Fix plugins' references to options

owen-m1 committed:
#1514: Add conditional for rootEl

owen-m1 committed:
Improve OnSpill support with multiple sortables + fix drop events

owen-m1 committed:
Create LICENSE

owen-m1 committed:
plugin API revamp (#1595)

owen-m1 committed:
#1611: Remove MultiDrag events on GLOBAL destroy

owen-m1 committed:
#1597: Fix docs for enabling plugins

owen-m1 committed:
#1599: Support for transforms on ghost element

owen-m1 committed:
Animation performance improvements

petestuart committed:
Fix call to animate() (#1630)

owen-m1 committed:
#1634: Fix onSpill plugins on mobile

owen-m1 committed:
#1598: Allow selecting with SHIFT key without multiDragKey being down

owen-m1 committed:
1.10.0

zjffun committed:
disable delayed drag events during destory

pangjian committed:
Fix(utils): Chrome < 44 not support scrollingElement

owen-m1 committed:
#1640: Fix OnSpill destroy error

owen-m1 committed:
#1646: Fix SSR rendering

owen-m1 committed:
Do not throw error if imported in nodeJS context

owen-m1 committed:
Fix saucelabs logo for npm

owen-m1 committed:
1.10.1

owen-m1 committed:
Fix optionListeners typo

monkeywithacupcake committed:
Added Open Collective

ohepworthbell committed:
#1644: Remove transform on drop

owen-m1 committed:
disable delayed drag events during destory

disable delayed drag events during destory

jmfolds committed:
add MSCSSMatrix to matrixFn options

owen-m1 committed:
Merge pull request #1674 from jmfolds/ie11-multidrag-fix

add MSCSSMatrix to matrixFn options

cawabunga committed:
Fix clicking select tag

Commit:
Added type check for 'dragStarted' variable as this is causing thousands of console errors in Chrome, Firefox, Edge, IE, Safari before the first drag.
This change is harmless because it is just checking if the variable dragStarted is declared before using it.

Tested across all of the above browsers and working fine after this change.

owen-m1 committed:
Merge pull request #1689 from kcak11/master

Added type check for 'dragStarted' variable as this is causing thousa…

dependabot[bot] committed:
Bump mixin-deep from 1.3.1 to 1.3.2

Bumps [mixin-deep](https://github.com/jonschlinkert/mixin-deep) from 1.3.1 to 1.3.2.
- [Release notes](https://github.com/jonschlinkert/mixin-deep/releases)
- [Commits](https://github.com/jonschlinkert/mixin-deep/compare/1.3.1...1.3.2)

Signed-off-by: dependabot[bot] <support@github.com>

owen-m1 committed:
Merge pull request #1700 from SortableJS/dependabot/npm_and_yarn/mixin-deep-1.3.2

Bump mixin-deep from 1.3.1 to 1.3.2

Commit:
Update README.md

Adds Typescript definitions package in Features

owen-m1 committed:
Merge pull request #1705 from waynevanson/patch-1

Update README.md

owen-m1 committed:
Merge pull request #1647 from pangjian/master

Fix(utils): Chrome < 44 not support scrollingElement

owen-m1 committed:
#1699: Fixed multiplying ghost transform

owen-m1 committed:
1.10.2

Commit:
Minor fix for IE compatibility in st/app.js UI of thresholds demo

owen-m1 committed:
Merge pull request #1732 from xyx-is/master

Minor fix for IE compatibility in st/app.js UI of thresholds demo

owen-m1 committed:
update example

owen-m1 committed:
Merge pull request #1681 from cawabunga/master

Fix clicking select tag

ddenev committed:
Fixed a typo in the description of the 'closest' method

owen-m1 committed:
Merge pull request #1764 from drank/master

Fixed a typo in the description of the 'closest' method

thiagocorbalan reviewed (commented):
Acredito que precisa ser feito um Rebase, :D

thiagocorbalan commented:
Antes de mergear, só precisa resolver esses conflitos do JS :D

RomualdoGrillo committed:
Solves "inser Before" issue on nested lists 

That solves an issue with nested sortables.
example of error https://codepen.io/romualdogrillo/pen/jOPoRJe
An error appears when all following conditions are met:
1)You have two nested sortable containers
2)option “pull:clone” i set at least on the parent container
3)You drag an item from the parent container into the nested container

The item is moved but not cloned and you get the error: 
“Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted”

Commit:
creates new github issue templates

Commit:
adds message with links to issue and codesandbox

Commit:
removes old issue template

waynevanson committed:
Merge pull request #1793 from waynevanson/master

creates new github issue templates

Commit:
moves issue templates to the correct folder

waynevanson committed:
Merge pull request #1794 from waynevanson/master

moves issue templates to the correct folder

owen-m1 committed:
Merge pull request #1784 from RomualdoGrillo/master

Solves "insert Before error" issue on nested lists

alexzuza committed:
Not tree-shake repaint function by compressors like terser

Closes #1658

owen-m1 committed:
Merge pull request #1834 from alexzuza/patch-1

Not tree-shake repaint function by compressors like terser

owen-m1 committed:
Error when mounting duplicate plugins

owen-m1 committed:
Fix #1904

owen-m1 committed:
#1878: fix onMove docs

owen-m1 committed:
Improve AutoScroll docs

owen-m1 committed:
Further improve AutoScroll docs

owen-m1 committed:
#1571: Dont use pointer events in Safari

owen-m1 committed:
#1954: Add useAnimation option to sort()

owen-m1 committed:
1.13.0

Commit:
Revert "Redo async-define adjustment and update minified version"

This reverts commit 7082daff9ab51d4ca8f7a9686fd7890684329051.

Commit:
Revert "Elo7 async define friendly"

This reverts commit b54bc0e5ae882e8f499d417c03843cd183ec53fb.

Commit:
Merge remote-tracking branch 'base/master' into update-base

Commit:
Redo async-define adjustment and update minified version
